net/instaweb/rewriter/public/webscale_make_scripts_async.h

```diff
@@ -21,13 +21,20 @@ class HtmlElement;
 class MessageHandler;
 class RewriteDriver;
 
+// Filter to async custom third-party scripts.
 class WebscaleMakeScriptsAsync : public CommonFilter {
  public:
-  explicit WebscaleMakeScriptsAsync(RewriteDriver* rewrite_driver, MessageHandler* handler);
+  explicit WebscaleMakeScriptsAsync(RewriteDriver* rewrite_driver);
   virtual ~WebscaleMakeScriptsAsync();
 
+  // statistics is not really used. The comments in
+  // net/instaweb/rewriter/rewrite_driver.cc mentions that it is good for all
+  // new filters to export statistics. If it does, it should be added
+  // InitStats() else it breaks under Apache.
```

Comment:
This comment was in rewrite_driver.cc.

The sentence was missing a 'to' before InitStats().